sdk/freertos_app_cpu0/.project

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>freertos_app_cpu0</name>
+	<comment>Created by SDK v2019.1. amdc_freertos_bsp_cpu0 - ps7_cortexa9_0</comment>
+	<projects>
+		<project>amdc_freertos_bsp_cpu0</project>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.cdt.managedbuilder.core.genmakebuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.cdt.managedbuilder.core.ScannerConfigBuilder</name>
+			<triggers>full,incremental,</triggers>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.cdt.core.cnature</nature>
+		<nature>org.eclipse.cdt.managedbuilder.core.managedBuildNature</nature>
+		<nature>org.eclipse.cdt.managedbuilder.core.ScannerConfigNature</nature>
+	</natures>
+	<linkedResources>
+		<link>
+			<name>FreeRTOS-Kernel</name>
+			<type>2</type>
+			<location>S:/School/WEMPEC/firmware-rtos-e/sdk/FreeRTOS-Kernel</location>

```

Comment:
Absolute links (probably don't need to fix now, but eventually)

Reply:
Hmm... not sure what to do about these. As far as I can tell, I've set all of the references to be workspace-relative paths in the properties menu for each sdk project (and they appear that way in the `.cproject` file.) I could try manually editing these absolute paths in these `.project` files in each `freertos-app-cpuX`, but I'm not sure whether or not that will cause issues. 

I'll leave these unresolved for now.

Reply:
Let's keep this on the back burner